lib/stsci/tools/tests/test_bitmask.py

```diff
@@ -0,0 +1,186 @@
+"""
+A module containing unit tests for the `bitmask` modue.
+
+:Authors: Mihai Cara (contact: help@stsci.edu)
```

Comment:
This is probably unnecessary since `git blame` would point to you anyway, but if you feel strongly about leaving your name in, you may ignore this comment.